Source/MediaInfo/Image/File_Jpeg.cpp

```diff
@@ -207,6 +207,43 @@ string Jpeg2000_Rsiz(int16u Rsiz)
     }
 }
 
+string ICC_Tag(int32u Signature)
+{
+    switch (Signature)
+    {
+        case 0x63707274: return "Copyright";
+        case 0x64657363: return "Profile description";
+        case 0x77747074: return "White point";
+        case 0x626B7074: return "Black point";
+        case 0x72545243: return "Reproduction curve, red";
+        case 0x67545243: return "Reproduction curve, green";
+        case 0x62545243: return "Reproduction curve, blue";
+        case 0x7258595A: return "Matrix, red";
+        case 0x6758595A: return "Matrix, green";
+        case 0x6258595A: return "Matrix, blue";
+        default        : return Ztring().From_CC4(Signature).To_UTF8();
+    }
+}
+
+string ICC_ColorSpace(int32u ColorSpace)
+{
+    switch (ColorSpace)
+    {
+        case 0x434D5920: return "CMY";
+        case 0x434D594B: return "CMYK";
+        case 0x47524159: return "Y";

```

Comment:
GRAY?

Reply:
MediaInfo uses "Y" for gray in other parsers, using 1 letter per component (make things easier when e.g. you add "A" to this line).